lsr_role2collection/collection_readme.txt

```diff
@@ -0,0 +1,36 @@
+﻿# Linux System Roles Ansible Collection
+Ansible collection for interacting with the Linux System Roles.
+
+## Currently supported distributions
+* Fedora
+* Red Hat Enterprise Linux (RHEL 6+)
+* RHEL 6+ derivatives such as CentOS 6+
+
+## Support
+
+### Supported Ansible Versions
+The supported Ansible versions are aligned with currently maintained Ansible versions that support Collections (Ansible 2.9 and later). You can find the list of maintained Ansible versions [here](https://docs.ansible.com/ansible/latest/reference_appendices/release_and_maintenance.html#release-status).
+
+## Installation
+There are currently two ways to use the Linux System Roles Collection in your setup: install from Ansible Galaxy or via RPM.
+
+### Installation from Ansible Galaxy
+You can install the collection from Ansible Galaxy by running `ansible-galaxy collection install fedora.linux_system_roles`.
+
+After the installation, the roles are available as `fedora.linux_system_roles.<role_name>`.
+
+Please see the Using [Ansible collections documentation](https://docs.ansible.com/ansible/devel/user_guide/collections_using.html) for further details.
+
+### Installation via RPM
+
+You can install the collection with the software package management tool `dnf` by running `dnf install linux-system-roles`.

```

Comment:
I suggest also having the command in a separated line or box, to make it easier to find, like:
"You can install the collection with the software package management tool `dnf` by running:

$  `dnf install linux-system-roles`"

What do you think?